WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -116,7 +116,7 @@ def post_announcement
     end
     title = "#{comp.name} on #{date_range_str} in #{comp.cityName}, #{comp.countryId}"
 
-    body = "The [#{comp.name}](#{root_url}results/c.php?i=#{comp.id})"
+    body = "The [#{comp.name}](#{root_url(locale:nil)}results/c.php?i=#{comp.id})"
```

Comment:
Oh right, this is stored in the database. Indeed we need to find a way to localize these links too !